tap_dashboard_reports/streams.py

```diff
@@ -99,14 +104,11 @@ def _fetch_one_report(self, client, start_date, end_date):
             end_date=end_date,
             **self._report.get("vars", {})
```

Comment:
It thinks it can be more readable to place this logic here:
```
if len(self.primary_keys) != 0:
             result[self.primary_keys] = _format_date(end_date)
```

Reply:
move logic to def _fetch_one_report